### ISSUE TITLE
[bookmark] Make color of icon only once for a list of bookmarks

### ISSUE THREAD

**Issue by dpriskorn** [organicmaps/organicmaps#5622]
**Is your feature request related to a problem? Please describe.**
When creating a new bookmark I want it to automatically get the same color icon ad all the others in the same list
Now I have to remember and choose the right color each time which is annoying.

**Describe the solution you would like**
Make icon color a property of the list and don't make me think about it when creating a bookmark.

**Describe alternatives you have considered**

**Additional context**

**Comment by biodranik** [comment]
Now the next created bookmark should have the same color as a previous bookmark and should go into the same last used list. Doesn't it solve your issue?

**Comment by foersterp** [comment]
I've got the same problem. For example, when I have a list with places I want to go and another list with stuff I want to edit in OSM. Right now, I have to select the list, say OSM, and then the corresponding color. It would be handier if the color was chosen automatically. Then I want to add something to the other list. I have to choose the list and, unnecessarily, the color, manually again.

**Comment by d4f5409d** [comment]
I guess it should looks something like this:

Lists menu -> List 1, List 2
List 1 -> List settings -> Color -> Blue, Description -> Interesting
List 2 -> List settings -> Color -> Red, Description -> See it again

Saving bookmark 1 -> List selector dialog -> List1

The bookmark will appear as the previously configured one in the List settings (ex. Blue color and description "Interesting"

Saving bookmark 2 - List selector dialog -> List 2

Bookmark 2 has red color and the description "See it again".

**Comment by biodranik** [comment]
Complicated saving bookmark is not an option. Now it's done in one click. Doing it in more than one click breaks the UX. 

Always using list colors also breaks existing UX of using the last color/last list. 

Is there a solution that doesn't break the existing UX?

**Comment by d4f5409d** [comment]
> Complicated bookmark saving is not an option. Now it's done in one click. Doing it in more than one click breaks the UX. 
> 
> Always using list colors also breaks existing UX of using the last color/last list. 
> 
> Is there a solution that doesn't break the existing UX?

Don't know for sure, what I know sure that the current implementation is bad, that we have to select it everytime we create bookmarks if we have multiple lists. I don't think my suggestion would break UX that much, and I think others would also benefit from it and enjoy it.

**Comment by d4f5409d** [comment]
Talking about UX, what guidelines do you follow to decide whether it breaks it or not? @biodranik

**Comment by biodranik** [comment]
When millions of users are using it for many years, changing UX is not an easy thing. 

Current flow allows easily add bookmarks of the same color into the same list. To start adding bookmarks to a different list with a different color user needs to edit bookmark and change its list and color once.

**Comment by d4f5409d** [comment]
![Screenshot_2023-08-12-21-47-55-38_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/organicmaps/organicmaps/assets/76624594/3d180306-63e4-40b3-8314-640b86fe577a)

https://github.com/organicmaps/organicmaps/issues/1060

I am not sure why you are against it, and I am sure you are wrong about this one looking at the community feedback. I think OM is being held back by these decisions. Don't get me wrong, I am really glad about your effort and responsiveness when it comes to implementing new things, but let's try to be open. Maybe we just misunderstood each other (which is a common problem here sadly) and I will make sure to do a concept art to make sure you see what we mean by that feature.

**Comment by biodranik** [comment]
I've exactly described my concerns about breaking the existing UX. I'm not against new features. I'm against making product more complex to use for the majority of users. 
Please propose a balanced solution.

**Comment by d4f5409d** [comment]
I guess then let's wait for the concept art and we can continue after that because I have ideas

**Comment by biodranik** [comment]
Great! It would be nice to have all scenarios covered there.

**Comment by pastk** [comment]
A solution that doesn't break the current UX is described in https://github.com/organicmaps/organicmaps/issues/1805

Feel free to re-open this issue if you think its different.
